### PR TITLE
[DOCS] Threat intel data grouped under new "Enriched data" section on Overview tab

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/1464.

Preview [here](https://security-docs_1484.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#view-alert-details).

### CODE REVIEW

**Review by benironside — APPROVED** [review]
LGTM!

File: docs/detections/alerts-ui-manage.asciidoc [hunk 1]
```diff
@@ -70,22 +70,24 @@ The Alert details flyout contains the following informational tabs:
...
 image::images/reason-section.png[Shows the Reason section of an alert]
 
-The *Document Summary* section provides general alert details, including any available threat intelligence data. Use this information to investigate what generated the alert.
+The *Document Summary* section provides general alert details. Use this information to investigate what generated the alert.
 
 [role="screenshot"]
 image::images/document-summary-section.png[Shows the Document Summary section of an alert]
 
-{elastic-sec} collects threat intelligence data for all alerts. Fields that show threat intelligence data display an informational icon.
+The *Enriched data* section displays available threat indicator matches and threat intelligence data. Click the information icon to learn more about the type of data that was collected.
```
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:87]: A little more concise?
```suggestion
The *Enriched data* section displays available threat indicator matches and threat intelligence data. Click the information icon to learn more about the collected data.
```

**Review by joepeeples — APPROVED** [review]
LGTM, a single nit

**Comment by ecezalp** [comment]
looks like there is the screenshot for `Threat Match Detected`, there is also the other version which is `Enriched with Threat Intelligence`, we should probably add both and explain the difference between the two (or link to a document that does)

**Review by ecezalp — COMMENTED** [review]
looks like there is the screenshot for Threat Match Detected, there is also the other version which is Enriched with Threat Intelligence, we should probably add both and explain the difference between the two (or link to a document that does)

File: docs/detections/alerts-ui-manage.asciidoc [hunk 2]
```diff
@@ -70,22 +70,24 @@ The Alert details flyout contains the following informational tabs:
 
 The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to analyze what generated the alert, so you can triage its relevance and resolve it as appropriate.
 
-The *Overview* tab has two main sections: *Reason* and *Document Summary*. The *Reason* section describes what generated the alert and provides general alert details. Use this information to understand the alert's origin and determine if the alert is relevant to your investigation.
+The Overview tab has three main sections: *Reason*, *Document Summary*, and *Enriched data*. The *Reason* and *Document Summary* sections are standard for every alert; the *Enriched data* section is conditional and only displays for alerts with intelligence data.
```
Comment (jmikell821) [inline docs/detections/alerts-ui-manage.asciidoc:73]:
```suggestion
The *Overview* tab has three main sections: *Reason*, *Document Summary*, and *Enriched data*. The *Reason* and *Document Summary* sections are standard for every alert. The *Enriched data* section is conditional and only displays alerts with intelligence data.
```

File: docs/detections/alerts-ui-manage.asciidoc [hunk 3]
```diff
@@ -70,22 +70,24 @@ The Alert details flyout contains the following informational tabs:
...
 image::images/reason-section.png[Shows the Reason section of an alert]
 
-The *Document Summary* section provides general alert details, including any available threat intelligence data. Use this information to investigate what generated the alert.
+The *Document Summary* section provides general alert details. Use this information to investigate what generated the alert.
 
 [role="screenshot"]
 image::images/document-summary-section.png[Shows the Document Summary section of an alert]
 
-{elastic-sec} collects threat intelligence data for all alerts. Fields that show threat intelligence data display an informational icon.
+The *Enriched data* section displays available threat indicator matches and threat intelligence data. Click the information icon to learn more about the type of collected data.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/alerts-ui-manage.asciidoc:87]: Changing to info icon as per discussion in #ux-writing.

**Review by ecezalp — COMMENTED** [review]
LGTM - in addition to the screenshots, it might be useful to actually explain the difference between a threat match and an investigation time enrichment or link to the docs that explains that

**Review by ecezalp — APPROVED** [review]

File: docs/detections/alerts-ui-manage.asciidoc [hunk 4]
```diff
@@ -102,11 +104,21 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
...
 
 Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
 
-IMPORTANT: After upgrading to {stack} version 7.15.x from release versions 7.12.0 through 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to instructions for <<post-upgrade-req-cti-alerts, migrating detection alerts enriched with threat intelligence data>>.
+IMPORTANT: After upgrading to {stack} version 7.15.0 or higher from release versions 7.12.0 through 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to instructions for <<post-upgrade-req-cti-alerts, migrating detection alerts enriched with threat intelligence data>>.
```
Comment (jmikell821) [inline docs/detections/alerts-ui-manage.asciidoc:121]:
```suggestion
IMPORTANT: After upgrading to {stack} version 7.15.0 or newer from release versions 7.12.0 through 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to instructions for <<post-upgrade-req-cti-alerts, migrating detection alerts enriched with threat intelligence data>>.
```

File: docs/detections/alerts-ui-manage.asciidoc [hunk 5]
```diff
@@ -102,11 +104,21 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
+Discovered threat indicator match data is placed under the Threat Match Detected sub-section.
+
+[role="screenshot"]
+image::images/matched-indicator-sub-sec.png[Shows the accompanying informational message, 600]
+
+All other available threat intelligence data is placed under the Enriched with Threat Intelligence sub-section.
```
Comment (jmikell821) [inline docs/detections/alerts-ui-manage.asciidoc:112]: I would probably bold the section names. 
```suggestion
All other available threat intelligence data is placed under the **Enriched with Threat Intelligence** sub-section.
```

File: docs/detections/alerts-ui-manage.asciidoc [hunk 6]
```diff
@@ -102,11 +104,21 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 If these fields aren't available, {elastic-sec} does not perform the query and does not display threat intelligence data for the alert.
 
+Discovered threat indicator match data is placed under the Threat Match Detected sub-section.
```
Comment (jmikell821) [inline docs/detections/alerts-ui-manage.asciidoc:107]:
```suggestion
Discovered threat indicator match data is placed under the **Threat Match Detected** sub-section.
```

**Review by jmikell821 — COMMENTED** [review]
Just a few nits.

**Pull request merged**